UM/Controller.py

```diff
@@ -315,3 +316,38 @@ def getToolsEnabled(self):
 
     def setToolsEnabled(self, enabled):
         self._tools_enabled = enabled
+
+    # Rotate camer view according defined angle
+
+    last_rotation_angle_x = 0
+    last_rotation_angle_y = 0
+    def rotateView(self, cooridnate = "x", angle = 0):
+        camera = self._scene.getActiveCamera()
+        camera_tool = self.getTool("CameraTool")
```

Comment:
how about self._camera_tool?

Reply:
This would bite us in the bun-buns if we're going to have multiple cameras, such as for a print overview or a preview icon in the g-code file. Fixed in master.